src/compatTest/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPluginTests.kt

```diff
@@ -55,7 +58,7 @@ class NexusPublishPluginTests {
 
     private val gson = Gson()
 
-    private val gradleVersion = System.getProperty("compat.gradle.version")
+    private val gradleVersion = System.getProperty("compat.gradle.version") ?: GradleVersion.current().version
```

Comment:
❓ This should always be set since we only run these via a `Test` task configured by Stutter or am I missing sth.?

Reply:
I'm running those tests from Idea itself (not Gradle from Idea) and then it doesn't seem to be set.

Reply:
Understood, would delegating test execution in IDEA work for you or is there a reason not to use that?

Reply:
Running test from Idea itself it faster and I try to use it by default (although Idea guys made it a little bit [harder](https://youtrack.jetbrains.com/issue/IDEA-223392)). While for the functional tests with TestKit it's negligible, I like to still have an option to do that (especially if just some minor changes are required for that).

Reply:
Ok, fair enough.